src/store/getters.js

```diff
@@ -94,6 +94,12 @@ export default {
     && (TX_TYPE_MDW[tx.type]
       || SCHEMA.OBJECT_ID_TX_TYPE[tx.tag]
       || (Object.values(SCHEMA.TX_TYPE).includes(tx.type) && tx.type)),
+
+  getInnerTransaction: (_, { getTxType }) => (tx) => [
+    TX_TYPE_MDW.GAMetaTx,
+    TX_TYPE_MDW.PayingForTx,
+  ].includes(getTxType(tx)) ? tx.tx?.tx : tx,
```

Comment:
Here's a PR that moves the `getTxType` from the store to the utils as it is not using the store state at all: https://github.com/aeternity/superhero-wallet/pull/1827

So if the `getTxType` is not using the store state same the `getInnerTransaction` can be moved into the `utils` file.

Reply:
Another thing that caught my eye is the name of the function - it suggest it will return the inner transaction. But:
1. It's inner tx
2. If the test return fail the regular tx is returned, so the function actually return inner or regular tx. My suggestion is to return `null` instead of the `tx` in this case.

Reply:
This function suppose to return the innerTx for the last nested transaction, maybe i should rename it to something more meaningful, `lastNestedInnerTx`.

Reply:
Sure, makes sense.